src/components/application_manager/src/resumption/resumption_data_processor.cc

```diff
@@ -55,7 +55,9 @@ ResumptionDataProcessor::ResumptionDataProcessor(
     : event_engine::EventObserver(application_manager.event_dispatcher())
     , application_manager_(application_manager) {}
 
-ResumptionDataProcessor::~ResumptionDataProcessor() {}
+ResumptionDataProcessor::~ResumptionDataProcessor() {
+  LOG4CXX_DEBUG(logger_, "destructing " << this);
```

Comment:
@mked-luxoft Do we need this log here? May be `LOG4CXX_AUTO_TRACE(logger_);` ?

Reply:
@ZhdanovP  removed in 23e9b6736a69f55e45d495d644b00c6437b37002